.changeset/chatty-wombats-buy.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/create-app': patch
+---
+
+removed inline and internal CSS from index.html
+
+To make this change to an existing app, apply the following changes to the `app/public/index.html` file:
```

Comment:
Path here is `packages/app/public/index.html`

Reply:
Done!